Szwalnia/ObslugaZamowien.cs

```diff
@@ -21,7 +21,23 @@ public ObslugaZamowien()
 
         private void ObslugaZamowien_FormClosed(object sender, FormClosedEventArgs e)
         {
-            Start.GetForm.Show();
+
+            if (Application.OpenForms.OfType<MagazynForm>().Count() > 0)
```

Comment:
jak chcemy się bawić w sprawdzanie itd to powinien to być statyczny konstruktor, tak jak mamy w StartForm